src/main/java/org/traccar/web/client/model/DataService.java

```diff
@@ -53,6 +53,8 @@
 
     Device updateDevice(Device device) throws TraccarException;
 
+    Device loadSensors(Device device);
```

Comment:
Why do you need a separate method to load sensors from device? Sensors collection is already present in device object. You can look how it is used in the `SensorsEditor` class.